vapi/simulator/simulator.go

```diff
@@ -382,11 +383,13 @@ func (s *handler) session(w http.ResponseWriter, r *http.Request) {
 		id = uuid.New().String()
 		now := time.Now()
 		s.Session[id] = &rest.Session{User: user, Created: now, LastAccessed: now}
-		http.SetCookie(w, &http.Cookie{
-			Name:  internal.SessionCookieName,
-			Value: id,
-			Path:  rest.Path,
-		})
+		if useHeaderAuthn != "true" {
```

Comment:
trying to understand when useHeaderAuthn would be false..It is set to true in line 147.

Reply:
vapi/simulator is part of 'vcsim', which is not limited to govmomi clients.